environments.ini

```diff
@@ -1539,6 +1539,43 @@ build_flags =
 custom_description = BLE gateway on the C3
 custom_hardware = AirM2M ESP32C3-CORE
 
+; Wemos Lolin C3 mini v2.3.0
+[env:lolin_c3_mini]
+;platform = ${com.esp32_c3_s3_platform} ; standard is @5.2.0, causes `assert rwble.c 261` error
+platform = espressif32@5.3.0  ; override to @5.3.0. Includes a fixed for the `assert rwble.c 261` error
+board = lolin_c3_mini
+board_build.partitions = min_spiffs.csv
+monitor_speed = 115200
+lib_deps =
+  ${com-esp.lib_deps}
+  ${libraries.wifimanager32}
+  ${libraries.ble}
+  ${libraries.decoder}
+build_flags =
+  ${com-esp.build_flags}
+  '-DZgatewayBT="BT"'
+; FastLED must be disabled. FastLED is not compatible with this board at the current moment.
+; The LED pins must still be defined, to avoid an error loop on pin definitions.
+; Available GPIOs are 0 - 8, 10, 20, 21.
+; See https://community.openmqttgateway.com/t/watchdog-kills-omg-on-esp32-c3-within-seconds/2192/15
+  '-DLED_SEND_RECEIVE=10'
+  '-DLED_INFO=6'
+  '-DLED_ERROR=7'
+  '-DTRIGGER_GPIO=9'
+  '-DNO_INT_TEMP_READING=true' ; No internal temperature on ESP32 C3 or S3
+  '-DGateway_Name="OpenMQTTGateway_ESP32C3_BLE"'
+custom_description = BLE gateway on ESP32C3
+custom_hardware = Wemos Lolin C3 mini v2.3.0
+
+; Serial output currently causes a delay on each message, if no serial listener is connected.
+; Use the _no_serial environment unless you are actively debugging.
+[env:lolin_c3_mini_no_serial]
```

Comment:
@Argafal for the moment I would merge the no serial into the environment above to avoid users going into the same issues as you

Reply:
Fine with me. How about we do it the other way round: make no_serial the default, but also provide a target for debugging with the serial output enabled. I am still keen on understanding the random reboot...

What are the default log levels for those two parameters? -DWM_LOG_LEVEL=? and -DLOG_LEVEL=?

Reply:
Make no_serial the default is perfect

For `LOG_LEVEL=` if you don't define it, it would be NOTICE
https://github.com/Legion2/OpenMQTTGateway/blob/257b39da8136448ee10ed4e39e7b3d0c086f73a5/main/User_config.h#L487

and NOTIFY for WM
https://github.com/Legion2/OpenMQTTGateway/blob/257b39da8136448ee10ed4e39e7b3d0c086f73a5/main/User_config.h#L130

Reply:
Perfect. Thanks for pointing out where to find this part in the documentation as well. Maybe I can understand the random reboot better by increasing the log levels in my next test run.

New patch below.